components/org.wso2.carbon.status.dashboard.core/pom.xml

```diff
@@ -185,15 +185,17 @@
     </build>
     <properties>
         <export.package>
-            org.wso2.carbon.status.dashboard.core.*
+            !org.wso2.carbon.status.dashboard.core.internal*
```

Comment:
@Yasara123 Seems like a there is missing comma at the end of this line. Please check and fix.

Reply:
Fixed in aa5fc154b95c726ceaffe6f697ef4c8842ab77db